<launch>

  <node pkg="ros_imc_broker" type="ros_imc_adapter_node" name="ros_imc_adapter_node" output="screen">
    <param name="system_name" value="sparus"/>
    <param name="system_type" value="uuv"/>
    <param name="system_imc_id" value="8193"/>
    <param name="enable_loopback" value="True"/>

    <!-- matching vehicles-defs ports: 6002 (default) -->
    <param name="udp_port" value="6002"/>

    <!--param name="udp_port_tries" value="10"/-->
    <!--param name="multicast_addr" value="224.0.75.69"/-->
    <!--param name="multicast_port" value="30100""/-->
    <!--param name="multicast_port_range" value="5"/-->

    <!-- should match the Neptus comm port: 6001 (default) -->
    <param name="static_destinations_addrs" value="127.0.0.1:6001"/>
  </node>

  <node pkg="turbot_imc_broker" type="turbot_imc_broker_node" name="turbot_imc_broker_node" output="screen">
    <param name="outdir" value="$(find turbot_imc_broker)/data"/>
    <param name="filename" value="rhodamine_data.csv"/>

    <!-- NED -->
    <param name="param_ned_lat" value="/navigator/ned_origin_lat"/>
    <param name="param_ned_lon" value="/navigator/ned_origin_lon"/>

    <remap from="~rhodamine" to="/cola2_sensors/rhodamine"/>
    <remap from="~nav_sts" to="/cola2_navigation/nav_sts"/>
  </node>
</launch>
